app/javascript/packs/application.js

```diff
@@ -8,3 +8,17 @@
 // layout file, like app/views/layouts/application.html.erb
 
 console.log('Hello World from Webpacker')
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+document.addEventListener('DOMContentLoaded', () => {
+  let reactElement = document.getElementById('testing')
+
+  if (reactElement) {
+    ReactDOM.render(
+      <h1>Boo yaa</h1>,
+      reactElement
+    )
```

Comment:
Um...you are not actually rendering your newly created react components. Please configure this T_T